modules/virt-connecting-vm-secondarynw-using-fqdn.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-connecting-vm-secondarynw-fqdn_{context}"]
+= Connecting to a virtual machine on a secondary network by using the cluster FQDN
+
+You can access a virtual machine (VM) that is attached to a secondary network interface from outside the cluster by using the fully qualified domain name (FQDN) of the cluster.
+
+.Prerequisites
+* The QEMU guest agent must be running on the virtual machine.
+* The IP address of the VM that you want to connect to, by using a DNS client, must be public.
+* You have configured the DNS server for secondary networks.
+* You have retrieved the fully qualified domain name (FQDN) of the cluster.
+
+.Procedure
+. Retrieve the virtual machine instance (VMI) configuration by using the following command:
++
+[source,terminal]
+----
+$ oc get vmi -n secondary-test vmi-sec -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
```

Comment:
Do we use VMI examples in our docs?

Reply:
No, so if there is a `VirtualMachine` manifest available, I can use that.

Reply:
@AlonaKaplan Can you please provide the `VirtualMachine` manifest for this step?

Reply:
https://github.com/openshift/openshift-docs/pull/56582/commits/2424d3496a36b3cbe56049c3731c8192824e479c#r1126946462